embulk-input-s3/src/test/java/org/embulk/input/s3/TestHttpProxy.java

```diff
@@ -48,7 +48,7 @@ public void checkHttpProxy()
             ConfigSource conf = config.deepCopy().set("http_proxy", httpProxyMap);
             S3PluginTask task = conf.loadConfig(S3PluginTask.class);
 
-            assertHttpProxy(new HttpProxy(host, Optional.<Integer>absent(), false, Optional.<String>absent(), Optional.<String>absent()),
+            assertHttpProxy(new HttpProxy(host, Optional.<Integer>absent(), Optional.<Boolean>absent(), Optional.<String>absent(), Optional.<String>absent()),
```

Comment:
Since the implementation was changed with `Task` interface, the unit tests were also changed. `false` case is here: https://github.com/embulk/embulk-input-s3/pull/52/commits/01d558d4f071e595b2458f730c5cdb2458406814#diff-b4f9d779f27fa3560e609a860446a29dR58